apps/src/applab/applab.js

```diff
@@ -560,14 +560,10 @@ Applab.init = function(config) {
     project.sourceHandler.setInitialLibrariesList(startLibraries);
     designMode.resetIds();
     Applab.setLevelHtml(config.level.startHtml || '');
-    let promise = Applab.storage.populateTable(level.dataTables, true); // overwrite = true
-    promise.catch(outputError);
-    Applab.storage.populateKeyValue(
-      level.dataProperties,
-      true,
-      () => {},
-      outputError
-    ); // overwrite = true
+    Applab.storage.clearAllData(
+      () => console.log('success'),
+      err => console.log(err)
+    );
```

Comment:
I see that you manually verified that initial table data is getting repopulated after you start over. How is that still working, given that you're removing this call to populateTable?

Reply:
Data is still getting repopulated because of this: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/applab/applab.js#L612

Under the old model, data didn't get deleted when you start over, but for levels with json-specified tables, we wanted to make sure the user was back to the starting state of the level, which is why we needed the call to `populateTable` with `overwrite=true`. Now, though, we clean up the data when you start over the level, so when we call `populateTable` with `overwrite=false` on page load, you get a fresh copy of the table.

Reply:
Ahh, nice catch and nice simplification! In that case, I would suggest taking this opportunity to remove the now-unused `overwrite` option too.